optaweb-tsp-planner/.gitignore

```diff
@@ -1,7 +1,7 @@
 !.mvn/wrapper/maven-wrapper.jar
 
 /target
-/local
```

Comment:
We should keep this. The `local/` dir should be still completely ignored.

Reply:
I've moved and changed /local with local/* otherwise 
```
local/openstreetmap/*
!local/openstreetmap/.gitkeep
```
won't work

Reply:
@yurloc find this on git docs:
[gitignore docs](https://git-scm.com/docs/gitignore/2.19.0#_pattern_format)

> An optional prefix "!" which negates the pattern; any matching file excluded by a previous pattern will become included again. **It is not possible to re-include a file if a parent directory of that file is excluded.** Git doesn’t list excluded directories for performance reasons, so any patterns on contained files have no effect, no matter where they are defined. Put a backslash ("\") in front of the first "!" for patterns that begin with a literal "!", for example, "\!important!.txt".

Make sense?